src/app/components/MediaLoader/index.tsx

```diff
@@ -169,7 +175,7 @@ const MediaLoader = ({ blocks, embedded, className }: Props) => {
   const config = buildConfig({
     id,
     blocks,
-    counterName,
+    counterName: pageIdentifierOverride || counterName,
```

Comment:
This might cause a conflict with Aaron's PR, but whoever merges first will have to fix it!